### PR TITLE
Some details fixed and new services added.

### PR DESCRIPTION
**Pull request by dessasin**
I corrected some of the deficiencies I saw in the default service settings and added a few new services, I did not have the chance to test. please check the changes before accepting

### CODE REVIEW

File: src/services/mastodon/service.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "id": "mastodon",
-  "name": "Mastodon",
-  "url": ""
-}
+    "id": "mastodon",
+    "name": "Mastodon",
+    "url": "https://mastodon.social/about"
```
Comment (sonnyp) [inline src/services/mastodon/service.json]: mastodon can be hosted anywhere, please revert this change

File: src/services/slack/service.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "id": "slack",
-  "name": "Slack",
-  "url": ""
-}
+    "id": "slack",
+    "name": "Slack",
+    "url": "https://slack.com/signin"
```
Comment (sonnyp) [inline src/services/slack/service.json]: yep that's probably better :+1:

File: src/services/trello/icon.svg [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="64pt" height="64pt" viewBox="0 0 64 64" version="1.1">
+<g id="surface1">
+<path style=" stroke:none;fill-rule:nonzero;fill:rgb(31.764706%,56.078431%,88.235294%);fill-opacity:1;" d="M 56.199219 0 L 7.800781 0 C 3.492188 0 0 3.492188 0 7.800781 L 0 56.199219 C 0 60.507812 3.492188 64 7.800781 64 L 56.199219 64 C 60.507812 64 64 60.507812 64 56.199219 L 64 7.800781 C 64 3.492188 60.507812 0 56.199219 0 Z M 27.839844 49.882812 C 27.839844 51.882812 26.21875 53.507812 24.214844 53.507812 L 11.945312 53.507812 C 9.941406 53.507812 8.320312 51.882812 8.320312 49.882812 L 8.320312 12.488281 C 8.320312 10.484375 9.941406 8.863281 11.945312 8.863281 L 24.214844 8.863281 C 26.21875 8.863281 27.839844 10.488281 27.839844 12.488281 Z M 55.679688 33.796875 C 55.679688 35.796875 54.058594 37.417969 52.054688 37.417969 L 39.785156 37.417969 C 37.78125 37.417969 36.160156 35.796875 36.160156 33.796875 L 36.160156 12.488281 C 36.160156 10.484375 37.78125 8.863281 39.785156 8.863281 L 52.054688 8.863281 C 54.058594 8.863281 55.679688 10.488281 55.679688 12.488281 Z M 55.679688 33.796875 "/>
+</g>
+</svg>
```
Comment (sonnyp) [inline src/services/trello/icon.svg]: where is the icon from?

File: src/services/outlook/icon.svg [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+<path style=" stroke:none;fill-rule:nonzero;fill:rgb(12.941176%,58.823529%,95.294118%);fill-opacity:1;" d="M 35.277344 2.460938 C 34.820312 2.082031 34.207031 1.917969 33.632812 2.039062 L 1.632812 8.039062 C 0.683594 8.214844 0 9.039062 0 10 L 0 54 C 0 54.960938 0.683594 55.789062 1.632812 55.964844 L 33.632812 61.964844 C 33.753906 61.988281 33.875 62 34 62 C 34.464844 62 34.914062 61.839844 35.277344 61.542969 C 35.734375 61.160156 36 60.59375 36 60 L 36 4 C 36 3.40625 35.734375 2.839844 35.277344 2.460938 Z M 35.277344 2.460938 "/>
+<path style=" stroke:none;fill-rule:nonzero;fill:rgb(98.039216%,98.039216%,98.039216%);fill-opacity:1;" d="M 18 46 C 12.488281 46 8 40.617188 8 34 C 8 27.386719 12.488281 22 18 22 C 23.511719 22 28 27.386719 28 34 C 28 40.617188 23.511719 46 18 46 Z M 18 26 C 14.691406 26 12 29.589844 12 34 C 12 38.414062 14.691406 42 18 42 C 21.308594 42 24 38.414062 24 34 C 24 29.589844 21.308594 26 18 26 Z M 18 26 "/>
+</g>
+</svg>
```
Comment (sonnyp) [inline src/services/outlook/icon.svg]: where is the icon from?

File: src/services/groupme/icon.svg [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+<path style=" stroke:none;fill-rule:nonzero;fill:rgb(84.313725%,91.372549%,100%);fill-opacity:1;" d="M 43.25 24.5 C 43.25 25.550781 42.425781 26.375 41.375 26.375 L 37.625 26.375 L 37.625 30.125 C 37.625 31.175781 36.800781 32 35.75 32 C 34.699219 32 33.875 31.175781 33.875 30.125 L 33.875 26.375 L 32 26.375 L 32 22.625 L 33.875 22.625 L 33.875 18.875 L 32 18.875 L 32 15.125 L 33.875 15.125 L 33.875 11.375 C 33.875 10.324219 34.699219 9.5 35.75 9.5 C 36.800781 9.5 37.625 10.324219 37.625 11.375 L 37.625 15.125 L 41.375 15.125 C 42.425781 15.125 43.25 15.949219 43.25 17 C 43.25 18.050781 42.425781 18.875 41.375 18.875 L 37.625 18.875 L 37.625 22.625 L 41.375 22.625 C 42.425781 22.625 43.25 23.449219 43.25 24.5 Z M 43.25 24.5 "/>
+<path style=" stroke:none;fill-rule:nonzero;fill:rgb(84.313725%,91.372549%,100%);fill-opacity:1;" d="M 48.574219 36.800781 C 44.863281 42.351562 38.675781 45.648438 32 45.648438 L 32 41.898438 C 37.398438 41.898438 42.425781 39.199219 45.425781 34.699219 C 46.023438 33.835938 47.1875 33.613281 48.050781 34.175781 C 48.914062 34.773438 49.136719 35.9375 48.574219 36.800781 Z M 48.574219 36.800781 "/>
+</g>
+</svg>
```
Comment (sonnyp) [inline src/services/groupme/icon.svg]: where is the icon from?

File: src/services/gmail/service.json [hunk 6]
```diff
@@ -0,0 +1,5 @@
+{
+    "id": "gmail",
+    "name": "Gmail",
+    "url": "https://accounts.google.com"
```
Comment (sonnyp) [inline src/services/gmail/service.json]: hmm? this URL does go to GMail

File: src/services/gmail/icon.svg [hunk 7]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="64pt" height="64pt" viewBox="0 0 64 64" version="1.1">
+<g id="surface1">
+<path style=" stroke:none;fill-rule:nonzero;fill:rgb(92.54902%,93.72549%,94.509804%);fill-opacity:1;" d="M 8 8 L 56 8 L 56 56 L 8 56 Z M 8 8 "/>
+<path style=" stroke:none;fill-rule:nonzero;fill:rgb(81.176471%,84.705882%,86.27451%);fill-opacity:1;" d="M 32 37.046875 L 56 56 L 56 18.585938 Z M 32 37.046875 "/>
+<path style=" stroke:none;fill-rule:nonzero;fill:rgb(95.686275%,26.27451%,21.176471%);fill-opacity:1;" d="M 58 8 L 56 8 L 32 26.953125 L 8 8 L 6 8 C 2.6875 8 0 10.6875 0 14 L 0 50 C 0 53.3125 2.6875 56 6 56 L 8 56 L 8 18.585938 L 32 37.042969 L 56 18.578125 L 56 56 L 58 56 C 61.3125 56 64 53.3125 64 50 L 64 14 C 64 10.6875 61.3125 8 58 8 Z M 58 8 "/>
+</g>
+</svg>
```
Comment (sonnyp) [inline src/services/gmail/icon.svg]: where is the icon from?

File: src/services/mastodon/service.json (hunk 1 above)
Comment (sonnyp) [reply src/services/mastodon/service.json]: ah ok mastodon.social is the main instance, makes sense since users can change the URL anyway

Comment (dessasin) [reply src/services/mastodon/service.json]: ok I will fix it

File: src/services/gmail/service.json (hunk 6 above)
Comment (dessasin) [reply src/services/gmail/service.json]: yes direct to gmail,have tried different browsers

**Comment by dessasin** [comment]
Btw icons from [Flaticon](https://www.flaticon.com/)

File: src/services/gmail/service.json (hunk 6 above)
Comment (sonnyp) [reply src/services/gmail/service.json]: ah right if you're not logged in already :+1:

**Comment by sonnyp** [comment]
> Btw icons from Flaticon

Please add proper credits to https://github.com/sonnyp/gigagram#credits in accordance with https://support.flaticon.com/hc/en-us/articles/207248209

**Comment by sonnyp** [comment]
Also, you can easily test your changes by doing

```sh
cd gigagram
./run.sh
```

**Comment by dessasin** [comment]
> Also, you can easily test your changes by doing
> 
> ```shell
> cd gigagram
> ./run.sh
> ```
run.sh can't run :/

`Gjs-Message: 21:15:57.246: JS LOG: Running from source tree, using local files

(dev.js:8754): Gjs-WARNING **: 21:15:57.354: JS ERROR: GLib.FileError: “/media/dessasin/Yedek/Work/Foss/gigagram/data/re.sonny.gigagram.services.gresource” dosyası açılamadı: open() başarısızlığı: Böyle bir dosya ya da dizin yok
register@/media/dessasin/Yedek/Work/Foss/gigagram/src/serviceManager.js:26:22
@/media/dessasin/Yedek/Work/Foss/gigagram/src/serviceManager.js:31:3
@/media/dessasin/Yedek/Work/Foss/gigagram/src/serviceManager.js:1:1
@/media/dessasin/Yedek/Work/Foss/gigagram/src/homePage.js:16:9
@/media/dessasin/Yedek/Work/Foss/gigagram/src/homePage.js:1:1
@/media/dessasin/Yedek/Work/Foss/gigagram/src/window.js:14:9
@/media/dessasin/Yedek/Work/Foss/gigagram/src/window.js:1:1
@/media/dessasin/Yedek/Work/Foss/gigagram/src/main.js:14:9
@/media/dessasin/Yedek/Work/Foss/gigagram/src/main.js:1:1
@./src/dev.js:9:1

Script ./src/dev.js threw an exception
`

**Comment by sonnyp** [comment]
there should be more logs before `Gjs-Message: 21:15:57.246: JS LOG: Running from source tree, using local files`

**Comment by dessasin** [comment]
> there should be more logs before `Gjs-Message: 21:15:57.246: JS LOG: Running from source tree, using local files`

```
./run.sh: 6: ./run.sh: glib-compile-resources: not found
Gjs-Message: 21:22:49.391: JS LOG: Running from source tree, using local files

```

**Comment by sonnyp** [comment]
@dessasin you need to install libglib2.0-dev or libglib2.0 (not sure)

**Comment by sonnyp** [comment]
also please `git pull`, I fixed an issue with the flatpak

**Comment by dessasin** [comment]
> also please `git pull`, I fixed an issue with the flatpak

yes run file worked :fireworks:

**Comment by dessasin** [comment]
And everything works perfectly

**Comment by sonnyp** [comment]
@dessasin could you add the ubuntu package to install in https://github.com/sonnyp/gigagram#install-development-dependencies and https://github.com/sonnyp/gigagram/pull/4#issuecomment-511142151 ?

I'll merge then.

**Comment by dessasin** [comment]
> @dessasin could you add the ubuntu package to install in https://github.com/sonnyp/gigagram#install-development-dependencies and [#4 (comment)](https://github.com/sonnyp/gigagram/pull/4#issuecomment-511142151) ?
> 
> I'll merge then.

I don't know how to do ubuntu packaging, but I can learn and add it.

**Comment by sonnyp** [comment]
@dessasin ah no, I meant just adding

`sudo apt install {WHAT_EVER_YOU_INSTALLED_FOR_GLIB}`

to https://github.com/sonnyp/gigagram#install-development-dependencies

**Comment by sonnyp** [comment]
Please feel free to add yourself to the list of contributors.
https://github.com/sonnyp/gigagram/edit/master/src/main.js#L66
